RxStarscreamTests/RxStarscreamTests.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  RxStarscreamTests.swift
+//  RxStarscreamTests
+//
+//  Created by Cezary Kopacz on 25/03/2017.
+//  Copyright © 2017 Guy Kahlon. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+import Starscream
+import RxStarscream
+
+extension WebSocketEvent: Equatable { }
+
+public func ==(lhs: WebSocketEvent, rhs: WebSocketEvent) -> Bool {
+    switch (lhs, rhs) {
+    case (.connected, .connected):
+        return true
+    case (.disconnected(_), .disconnected(_)):
+        return true
+    case (.message(_), .message(_)):
+        return true
+    case (.data(_), .data(_)):
+        return true
+    case (.pong, .pong):
+        return true
+    default:
+        return false
+    }
+}
+
+class RxStarscreamTests: XCTestCase {
+
+    private var connectedObserver: TestableObserver<Bool>!
+    private var pongObserver: TestableObserver<WebSocketEvent>!
+    private var responseObserver: TestableObserver<WebSocketEvent>!
+    private var socket: WebSocket!
+
+    let disposeBag = DisposeBag()
+
+    override func setUp() {
+        super.setUp()
+
+        socket = WebSocket(url: URL(string: "wss://echo.websocket.org")!)
+    }
+
+    func testConnection() {
+        socket.connect()
+        let connectExpectation = expectation(description: "Connect")
+        let scheduler = TestScheduler(initialClock: 0)
+
+        connectedObserver = scheduler.createObserver(Bool.self)
+
+        let connected = socket.rx.connected.shareReplay(1)
+
+        connected.subscribe(onNext: { [unowned self] _ in
+                    self.socket.disconnect()
+                }).addDisposableTo(disposeBag)
```

Comment:
As @freak4pc points in "sibling" PRs, that could be replaced with `dispose(by: disposeBag)` as `addDisposableTo` is going to be deprecated.